apps/finance/test/finance.js

```diff
@@ -13,10 +16,13 @@ contract('Finance App', accounts => {
     const periodDuration = 100
     const withdrawAddr = '0x0000000000000000000000000000000000001234'
 
+    const ETH='0x0'
```

Comment:
Can we get this from Finance as `Finance.ETH()` (if we made that public)?

Reply:
I tried, but it doesn't work for me from Truffle. I get:
```
TypeError: Finance.ETH is not a function
```